File: DogeFriendsApi/Dto/DogDto.cs
namespace DogeFriendsApi.Dto
{
    public class DogDto
    {
        public int Id { get; set; }
        public string Name { get; set; } = string.Empty;
        public int DogBreedId { get; set; }
        public string DogBreed { get; set; } = string.Empty;
        public string DogUser { get; set; } = string.Empty;
        public string DogUsername { get; set; } = string.Empty;
        public string Base64Image { get; set; } = string.Empty;
        public string ExternalId { get; set; } = string.Empty;
    }
}

File: AngularDogeFriendsClient/src/app/models/DogDto.ts
import { Images } from "./Images";

export interface DogDto {
  id: number;
  name: string;
  dogBreedId: number;
  dogBreed: string;
  dogUser: string;
  dogUsername: string;
  base64Image: string;
  externalId: string;
}

export interface DogDtoDetailed extends DogDto {
  images: Images
  coat: string,
  size: string,
  breedDescription: string,
  breedGroups: string
}
